osu.Game/Users/UserStatistics.cs

```diff
@@ -43,6 +43,9 @@ private int rank
         [JsonProperty(@"hit_accuracy")]
         public decimal Accuracy;
 
+        [JsonIgnore]
+        public string DisplayAccuracy => $"{Accuracy:0.00%}";
```

Comment:
This should probably match the same implementation as the other one. At which point it may be best to call an extension method for formatting in both cases (but leave the `DisplayAccuracy` properties for accessibility).

Reply:
Should I put the extension method under `osu.Game.Extensions`?

Reply:
`osu.Game.Utils`